### PR TITLE
Alpha Release for easy_manipulation_deployment

### PR DESCRIPTION
**Pull request by tanjpg**
This pull request is to upload the alpha release of easy_manipulation_deployment.

The codebase code quality standard is verified with tests conducted with industrial_ci and provides detailed documentation including step by step tutorials for new users to follow.

### CODE REVIEW

**Comment by cardboardcode** [comment]
The code under this pull request has been verified via internal peer review under the ROSI-AP team.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  CODE_COVERAGE: "codecov.io"
+  CODECOV_TOKEN: "36cef827-7894-4feb-b0d5-5c5c133cb341"
+
+foxy-moveit2:
+  variables:
+    ROS_DISTRO: "foxy"
+    DOCKER_IMAGE: "registry.gitlab.com/rosi-ap/rosiap_docker_images:moveit-foxy-nightly"
+    AFTER_INIT_EMBED: "source /opt/moveit2/install/setup.bash"
+  script: 
+    - echo "tLqbxEsTsUwtbjjwdBMd" | docker login -u rosi --password-stdin registry.gitlab.com
```
Comment (gavanderhoorn) [inline .gitlab-ci.yml:34]: Was this supposed to be merged in here? Or is that not a secret?

Comment (cardboardcode) [reply .gitlab-ci.yml:34]: Thanks, @gavanderhoorn. 

Noted on the requested change. @tanjpg Please make the appropriate amendment and update the pull request.

Comment (gavanderhoorn) [reply .gitlab-ci.yml:34]: Seeing as you've merged it, I don't believe you can update this PR any more.

That secret is in a few more places.

You should change the password on your `registry.gitlab.com` account as well I believe.

Comment (cardboardcode) [reply .gitlab-ci.yml:34]: @gavanderhoorn 

Apologies for this oversight. 

I have invalidated the password highlighted in this requested change. Will run another run of code review to address any further potential oversights before making a subsequent pull request. 

Thank you for pointing this out promptly. Will update once the new round of code review has completed.

Comment (gavanderhoorn) [reply .gitlab-ci.yml:34]: No problem, and no need to apologise.

Comment (Briancbn) [reply .gitlab-ci.yml:34]: > That secret is in a few more places.
> 
> You should change the password on your `registry.gitlab.com` account as well I believe.

Hi @gavanderhoorn, thanks for pointing it out. This is actually a deployment token from gitlab that only have read access to the gitlab container registry, but not the source code or any write access. It is not a password to our repository. That said, I will move the ci docker to the registry under Github this Github repo.

Comment (gavanderhoorn) [reply .gitlab-ci.yml:34]: At least Github Actions has a nice way to pass secrets such as this one to CI scripts (even though it only allows read-only access, it's still a "secret"). Travis should have that as well (but has other problems at the moment.

You may want to look into those.

**Pull request merged**